spec/barbeque/executor_spec.rb

```diff
@@ -19,11 +19,22 @@
     end
 
     it 'performs a job' do
-      expect(test_job).to receive(:perform).with(*args)
+      expect(test_job).to receive(:perform).with('foo', 'bar')
```

Comment:
Sorry, I don't get your comment well.
This fix is for consistency with L30. Did you mean we should not use `args` variable if we write argument directly?